lean4-infoview/src/infoview/info.tsx

```diff
@@ -121,10 +122,29 @@ export function InfoDisplay(props0: InfoDisplayProps) {
     });
 
     const rs = React.useContext(RpcContext);
+
+    // # widgets
+    const [widgetTrig, setWidgetTrig] = React.useState(0)
+    useClientNotificationEffect('textDocument/didChange', () => setWidgetTrig(widgetTrig + 1), [widgetTrig])
```

Comment:
This feels nonstandard

Reply:
Indeed.  More importantly, the widget should be part of the "pausable" "display props" which are set in `InfoAux`.  The widget request should be analogous to the goal request.  And certainly not separately triggered on didChange events.

Reply:
This is fixed now